sdk/eventhub/event-hubs/samples/sendEvents.ts

```diff
@@ -33,6 +32,7 @@ async function main(): Promise<void> {
   const client = new EventHubClient(connectionString, eventHubName);
   const partitionIds = await client.getPartitionIds();
   const producer = client.createProducer({ partitionId: partitionIds[0] });
+  const eventDatabatch = await producer.createBatch();
```

Comment:
Move this to just before the loop where we add events. That way user would know exactly what is needed to create the batch